boards/arm/sam_v71_xult/sam_v71_xult.dts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2019 Gerson Fernando Budke
+ * Copyright (c) 2017 Piotr Mienkowski
+ * Copyright (c) 2017 Justin Watson
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <atmel/samv71q21.dtsi>
+
+/ {
+	model = "Atmel SAM V71 Xplained Ultra board";
+	compatible = "atmel,sam_v71_xult", "atmel,samv71q21", "atmel,samv71";
+
+	aliases {
+		i2c-0 = &i2c0;
+		i2c-1 = &i2c1;
+		i2c-2 = &i2c2;
+		led0 = &yellow_led0;
+		led1 = &yellow_led1;
+		sw0 = &sw0_user_button;
+		sw1 = &sw1_user_button;
+	};
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		yellow_led0: led_0 {
+			gpios = <&porta 23 0>;
+			label = "User LED";
+		};
+		yellow_led1: led_1 {
+			gpios = <&portc 9 0>;
+			label = "User LED";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+
+		/* The switch is labeled SW300/301 in the schematic, and
+		 * labeled SW0 on the board, and labeld ERASE User Button
+		 * on docs
+		 */
+		sw0_user_button: button_1 {
+			label = "User Button";
+			gpios = <&porta 9 (GPIO_PUD_PULL_UP |
+					    GPIO_INT_ACTIVE_LOW)>;
+		};
+		sw1_user_button: button_2 {
+			label = "User Button";
+			gpios = <&portb 12 (GPIO_PUD_PULL_UP |
+					    GPIO_INT_ACTIVE_LOW)>;
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <300000000>;
+};
+
+&adc0 {
+	status = "okay";
+};
+
+&adc1 {
+	status = "okay";
+};
+
+&i2c0 {
+	status = "okay";
+};
+
+&i2c2 {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+};
+
+&usart1 {
+	current-speed = <115200>;
+	status = "okay";
+};
+
+&wdog {
+	status = "okay";
+};
+
+&usbhs {
+	status = "okay";
+};
+
+&pwm0 {
+	status = "okay";
+};
+
+&porta {
+	status = "okay";
+};
+
+&portb {
+	status = "okay";
+};
+
+&portc {
+	status = "okay";
+};
+
+&portd {
+	status = "okay";
+};
+
+&porte {
+	status = "okay";
+};
+
+&flash0 {
+	/*
+	 * If chosen's zephyr,code-partition
+	 * is unset, the image will be linked
+	 * into the entire flash device.  If
+	 * it points to an individual
+	 * partition, the code will be linked
+	 * to, and restricted to that
+	 * partition.
+	 */
```

Comment:
This has been copied from the current tree, however the description is not valid anymore #13151. I propose to simply remove it.